frontend/app/controllers/project/show/transportation.js

```diff
@@ -0,0 +1,7 @@
+import Controller from '@ember/controller';
+import { alias } from '@ember-decorators/object/computed';
+
+export default class ProjectShowTransportationController extends Controller {
+  @alias('model.project') project;
+  @alias('model.transportationAnalysis') analysis;
```

Comment:
I think `model.transportationAnalysis` is clearer directly in the template. Using an alias sorta conceals what's happening tbh... just thinking about a person coming in and learning the code.

For example, if it's passed as `model.transportationAnalysis`, we are crystal clear about the shape of this special object and where it's coming from.

Reply:
copied this from public-schools controller, to my mind the 'analysis' alias is not really obfuscating  but could go either way as long as both controllers match